src/hooks/useAggregatorAPR.ts

```diff
@@ -6,6 +6,7 @@ interface APRResponse {
     id: string
     chain_id: number
     is_farm: boolean
+    type: string
```

Comment:
better to specify it by its enum, e.g: 
```
    type: 'classic' | 'elastic'
```

Reply:
Yeah, it seems better. Thanks for your suggestion.
I see that we have `VERSION` constant. What do you think if I change the code above to something like:
```ts
import { VERSION } from 'constants/v2'

// ...and update the type
  type?: VERSION.CLASSIC | VERSION.ELASTIC
```

Reply:
That's even better